commons/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="fr.free.nrw.commons"
+    package="free.nrw.commons"
```

Comment:
I have deleted and regenerated the R file several times, but it does not appear to work. I consistently get this error: `Error:android-apt-compiler: [commons] invalid resource directory name: C:\Users\misao\Documents\GitHub\apps-android-commons\commons\res java`

I have also invalidated the cache and restarted IntelliJ, and checked for any false android.R imports that might block regeneration.